util.js

```diff
@@ -1,5 +1,5 @@
 const cookie = require('cookie')
-const yargs = require('yargs')
+const yargsParser = require('yargs-parser')
```

Comment:
Besides have a different API that chains method calls instead of passing an object, I don't understand what yargs actually does for us over plain yargs-parser, so I removed it.